### PR TITLE
Add input parameter to HelloIntelliJ program

### PR DESCRIPTION
**Pull request by moinnbr**
Fantastic Training.

### CODE REVIEW

File: src/HelloIntelliJ.java [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
     public static void main(String[] args) {
-        System.out.println("Hello IntelliJ! This is Moin");
-    }
-}
-
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter Your Name:");
+        String message = scanner.nextLine();
+        // add line to display message
+// add line to close scanner }}
```
Comment (DavPoghosyan) [inline src/HelloIntelliJ.java]: Please remove comments

File: src/HelloIntelliJ.java [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+        scanner.close();
+        System.out.println(message);
```
Comment (DavPoghosyan) [inline src/HelloIntelliJ.java:7]: please have proper order of commands, 
close should be at the end

**Review by DavPoghosyan — CHANGES_REQUESTED** [review]

File: src/HelloIntelliJ.java (hunk 1 above)
Comment (moinnbr) [reply src/HelloIntelliJ.java]: Done

File: src/HelloIntelliJ.java (hunk 2 above)
Comment (moinnbr) [reply src/HelloIntelliJ.java:7]: Done. Thank You.

Comment (DavPoghosyan) [reply src/HelloIntelliJ.java:7]: its ok, thanks

**Review by DavPoghosyan — APPROVED** [review]

**Pull request merged**